resources/assets/less/bem/supporter-promo.less

```diff
@@ -0,0 +1,90 @@
+/**
+ *    Copyright 2015-2018 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.supporter-promo {
+  display: flex;
+  justify-content: space-between;
+
+  padding-left: 65px;
+  padding-right: 75px;
+  height: 200px;
```

Comment:
mobile should look more or less like this

![](https://s.myconan.net/2018-02/firefox_2018-02-20_21-44-28.png)